server/jetstream_limits_test.go

```diff
@@ -0,0 +1,67 @@
+package server
+
+import (
+	"crypto/rand"
+	"testing"
+)
+
+func TestJetstreamAccountFileStoreLimits(t *testing.T) {
```

Comment:
```suggestion
func TestJetStreamAccountFileStoreLimits(t *testing.T) {
```
we should move this one to jetstream_cluster_3_test.go below the TestJetStreamClusterLimitsBasedStreamFileStoreDesync test which is related